src/components/Comments/Comment.js

```diff
@@ -206,6 +287,7 @@ const Comment = ({
       </View>
 
       <BlockComponent ref={refBlockComponent} screen={'feed_comment_item'} />
+      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component is added to the render method. It uses the `refSheet` ref and `dataSheet` array defined earlier. No issues found here.
```diff
+ <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->